forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-gateway/forgerock-openbanking-uk-aspsp-rs-gateway-server/src/test/java/com/forgerock/openbanking/aspsp/rs/api/payment/verifier/DetachedJwsVerifierTest.java

```diff
@@ -120,18 +121,14 @@ public void shouldFailToVerifyB64HeaderGivenVersionBefore3_1_4AndB64HeaderIsTrue
         assertThat(exception).hasMessage("Invalid detached signature " + detachedJws + ". Reason: b64 claim header not set to false");
     }
 
-    @Test
-    public void shouldFailToVerifyB64HeaderGivenVersionBefore3_1_4AndB64HeaderIsNonBoolean() throws Exception {
+    @Test(expected = OBErrorException.class)
```

Comment:
I'm not a fan of this approach as the correct exception could be thrown, but for the wrong reason.

Reply:
Also, it's now inconsistent with `catchThrowableOfType` used elsewhere.